subsys/storage/flash_map/flash_map_default.c

```diff
@@ -10,10 +10,48 @@
 #include <zephyr.h>
 #include <storage/flash_map.h>
 
-#define FLASH_AREA_FOO(part)						\
```

Comment:
e571c885653830bc643e27f1e920018a1241d6e6 reworks what goes from zephyr vanilla, this need to be reflected in
`[nrf noup] tree-wide: support NCS Partition Manager (PM) definitions` commit

Reply:
The NCS commit should just introduce the alternative inside for `#if USE_PARTITION_MANAGER` case

Reply:
fixed, please recheck